common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -60,6 +60,14 @@ export const commandInfo = makeCommandInfo(
   },
 );
 
+async function commitChanges(projectFolder: string, message: string): Promise<void> {
+  log.info("Committing changes, message: ", message);
+  await run(["git", "add", "."], { cwd: projectFolder });
```

Comment:
would some of the git helpers in `common\tools\dev-tool\src\util\git.ts` help?

Reply:
I looked into it, thank you! But I think what we have here is probably fine unless you feel strongly. It'd require some additional work to get `allow-empty` added to git.ts. Not a lot of work, but probably not worth the effort